doc/content/getting_started/installation.md

```diff
@@ -6,7 +6,7 @@
 In order to install TMAP8, the MOOSE developer environment must be installed. The
 installation procedure depends on your operating system, so click on the MOOSE
 website link below that corresponds to your operation system/platform and follow
-the instructions to the conda installation step named "Cloning MOOSE". Then,
+the instructions until you are done with the step named "Install MOOSE" (note that you do not need to follow the steps from the section titled "Cloning MOOSE" and below). Then,
```

Comment:
```suggestion
the instructions until you are done with the step named "Install MOOSE" (note that 
you do not need to follow the steps from the section titled "Cloning MOOSE" and below). Then,
```

Marking a line-length/formatting change to also let you know one that GitHub wouldn't let me mark. Could you fix "operation system" on the line above and make it "operating system" while we're here? Thanks!